src/App.js

```diff
@@ -1,30 +1,24 @@
 import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import SignIn from './pages/SignIn';
+import SignUp from './pages/SignUp';
+import Footer from './components/Footer';
+import Header from './components/Header';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
+import HomePage from './pages/HomePage';
 
-const URL = 'http://localhost:8000/api/v1/';
+//const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
-
-const [message, setMessage] = useState(''); 
-
-  useEffect(() => {
-
-    (async () => {
-      const myData = await getAllData(URL)
-      setMessage(myData.data);
-    })();
-      
-    return () => {
-      console.log('unmounting');
-    }
-
-  }, []);
-
   return (
-    <>
-      <h1>{message}</h1>
-    </>
+    <BrowserRouter>
+      <Header />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/signIn" element={<SignIn />} />
+        <Route path="/signUp" element={<SignUp />} />
+      </Routes>
+      <Footer />
+    </BrowserRouter>
```

Comment:
To my earlier point, it will be much more of a task to modify code for the backend API if we're already set up to use `react-router-dom` throughout the project.